tools/install.py

```diff
@@ -37,10 +37,11 @@
 
 
 def add_revision_file(target):
-  # text=True would be better than encoding here, but it's only supported in 3.7+
-  git_hash = subprocess.check_output(['git', 'rev-parse', 'HEAD'], encoding='utf-8').strip()
-  with open(os.path.join(target, 'emscripten-revision.txt'), 'w') as f:
-    f.write(git_hash + '\n')
+  if os.path.isdir('.git'):
```

Comment:
How about putting this condition around the callsite instead?  otherwise this function becoming more like "maybe_add_revision_file".

Reply:
Thank you for looking at this pull request.
I moved the the condition and added a comment.